mm2src/adex_cli/src/helpers.rs

```diff
@@ -4,11 +4,12 @@ use serde::{Deserialize, Serialize};
 use std::fs;
 use std::io::Write;
 use std::ops::Deref;
+#[cfg(unix)] use std::os::unix::fs::PermissionsExt;
 use std::path::Path;
 
 use crate::error_anyhow;
 
-pub(crate) fn rewrite_data_file<T>(data: T, file: &str) -> Result<()>
+pub(crate) fn rewrite_data_file<T>(data: T, file: &str, _unix_mode: Option<u32>) -> Result<()>
```

Comment:
I think `_unix_mode: Option<u32>` is unnecessary here. You can use `CFG_FILE_PERM_MODE` directly if `#[cfg(unix)]`

Reply:
I would not introduce business concept CFG_FILE_PERM_MODE right here in the common facilities module - to not mixing intentions.
What if I suggest the following changes, @ozkanonur?

https://github.com/KomodoPlatform/komodo-defi-framework/pull/1913/files

Reply:
It's same, my point was avoiding that overhad

Reply:
Thank you for the fast feedback!
I'm really sorry for misunderstanding, can I ask you to clarify please?
As I see you want me to put `CFG_FILE_PERM_MODE` that is not related to file processing layer right into the `helpers.rs`?

Is the following code what you want me to do?
```
pub(crate) fn set_file_permissions(file: &str) -> Result<()> {
    let mut perms = fs::metadata(file)?.permissions();
    perms.set_mode(CFG_FILE_PERM_MODE);
    fs::set_permissions(file, perms)?;
    Ok(())
}
```